sample/XUIComponents/UIComponents/UIComponents.Tizen.Wearable/UIComponents.Tizen.Wearable.csproj

```diff
@@ -1,33 +1,34 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Tizen.NET.Sdk/1.0.9">
+
 
-  <!-- Property Group for Tizen40 Project -->
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>tizen40</TargetFramework>
+    <!--<RuntimeIdentifier>tizen-armel</RuntimeIdentifier>-->
+    <CopyLocalRuntimeTargetAssets>true</CopyLocalRuntimeTargetAssets>
```

Comment:
Why did you add `CopyLocalRuntimeTargetAssets` ? 
Does this sample have any library wich is included in sample app tpk?

Reply:
I have copied from my other project. there should be copy paste mistake i will remove it.